### ISSUE TITLE
Game crashes after loading into world

### ISSUE THREAD

**Issue by tedrelish** [IrisShaders/Iris#1914]
### Minecraft Version

1.19.3

### Iris Version

iris-mc1.19.3-1.5.2

### Sodium Version

sodium-fabric-mc1.19.3-0.4.9+build.23

### Operating System

Windows 10

### What is your GPU?

Nvidia GForce GTX 1070 TI

### Java Version

Java 8 Update 361

### Reproduction Steps

1. Load game
2. Try and join server
3. Joins for a second or two and then my game crashes

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// Hi. I'm Minecraft, and I'm a crashaholic.
 
Time: 2023-03-14 13:45:52
Description: Unexpected error
 
java.util.NoSuchElementException
	at java.base/java.util.ArrayDeque.getLast(ArrayDeque.java:413)
	at net.minecraft.class_4587.method_22903(class_4587.java:532)
	at me.jellysquid.mods.sodium.client.render.SodiumWorldRenderer.renderTileEntities(SodiumWorldRenderer.java:258)
	at net.minecraft.class_761.handler$emb000$onRenderTileEntities(class_761.java:15605)
	at net.minecraft.class_761.method_22710(class_761.java:1364)
	at net.minecraft.class_757.method_3188(class_757.java:1085)
	at net.minecraft.class_757.method_3192(class_757.java:864)
	at net.minecraft.class_310.method_1523(class_310.java:1193)
	at net.minecraft.class_310.method_1514(class_310.java:781)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
 
 
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------
 
-- Head --
Thread: Render thread
Stacktrace:
	at java.base/java.util.ArrayDeque.getLast(ArrayDeque.java:413)
	at net.minecraft.class_4587.method_22903(class_4587.java:532)
	at me.jellysquid.mods.sodium.client.render.SodiumWorldRenderer.renderTileEntities(SodiumWorldRenderer.java:258)
	at net.minecraft.class_761.handler$emb000$onRenderTileEntities(class_761.java:15605)
	at net.minecraft.class_761.method_22710(class_761.java:1364)
	at net.minecraft.class_757.method_3188(class_757.java:1085)
 
-- Affected level --
Details:
	All players: 1 total; [class_746['ChipsNotFries'/1006, l='ClientLevel', x=-6421.95, y=77.00, z=4380.78]]
	Chunk stats: 1024, 729
	Level dimension: minecraft:overworld
	Level spawn location: World: (309,63,-463), Section: (at 5,15,1 in 19,3,-29; chunk contains blocks 304,-64,-464 to 319,319,-449), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,-64,-512 to 511,319,-1)
	Level time: 144352650 game time, 117109755 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:455)
	at net.minecraft.class_310.method_1587(class_310.java:2390)
	at net.minecraft.class_310.method_1514(class_310.java:805)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
 
-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, spoornpacks, Fabric Mods
 
-- System Details --
Details:
	Minecraft Version: 1.19.3
	Minecraft Version ID: 1.19.3
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1482073640 bytes (1413 MiB) / 6341787648 bytes (6048 MiB) up to 6341787648 bytes (6048 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 3600 6-Core Processor              
	Identifier: AuthenticAMD Family 23 Model 113 Stepping 0
	Microarchitecture: Zen 2
	Frequency (GHz): 3.59
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1070 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1b82
	Graphics card #0 versionInfo: DriverVersion=31.0.15.2686
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 16384.00
	Memory slot #2 clockSpeed (GHz): 2.13
	Memory slot #2 type: DDR4
	Virtual memory max (MB): 47716.59
	Virtual memory used (MB): 21871.73
	Swap memory total (MB): 15000.00
	Swap memory used (MB): 31.21
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx6048m -Xms256m
	Fabric Mods: 
		accelerateddecay: Accelerated Decay 2.0.0+mc1.19.3
		additionaladditions: Additional Additions 5.2.0
			libgui: LibGui 6.5.2+1.19.3
				jankson: Jankson 5.0.0+j1.2.1
				libninepatch: LibNinePatch 1.1.0
		additionalbars: Additional Bars 4.0.0+1.19.3
		adorn: Adorn 4.0.1+1.19.3
		advdebug: Advancements Debug 2.3.0
		ae2: Applied Energistics 2 13.0.6-beta
			noindium: No Indium? 1.1.0+1.19
		ae2wtlib: AE2WTLib 13.0.6-beta
		ambientenvironment: Ambient Environment 8.0+4
		appleskin: AppleSkin 2.4.2+mc1.19.3
		architectury: Architectury 7.1.70
		armorposer: Armor Poser 1.3.0
		backslot: BackSlot 1.2.12
			amecsapi: Amecs API 1.3.6+mc1.19.3
		badpackets: Bad Packets 0.3.4
		balm-fabric: Balm 5.0.3
		beautify: Beautify 1.1.0+fabric-1.19.3
		biomemakeover: Biome Makeover 1.19.3-1.7.1
			taniwha: Taniwha 1.19.3-3.6.5
		block_limit_fix: Block Limit Fix 1.0.3-fabric
		blockus: Blockus 2.6.3+1.19.3
			terraform-wood-api-v1: Terraform Wood API (v1) 5.0.0
		bookshelf: Bookshelf 17.0.5
		botanypots: BotanyPots 10.0.3
		botanypotstiers: BotanyPotsTiers 4.0.1
		botanytrees: BotanyTrees 6.0.1
		cardinal-components: Cardinal Components API 5.1.0
			cardinal-components-base: Cardinal Components API (base) 5.1.0
			cardinal-components-block: Cardinal Components API (blocks) 5.1.0
			cardinal-components-chunk: Cardinal Components API (chunks) 5.1.0
			cardinal-components-entity: Cardinal Components API (entities) 5.1.0
			cardinal-components-item: Cardinal Components API (items) 5.1.0
			cardinal-components-level: Cardinal Components API (world saves) 5.1.0
			cardinal-components-scoreboard: Cardinal Components API (scoreboard) 5.1.0
			cardinal-components-world: Cardinal Components API (worlds) 5.1.0
		carryon: Carry On 2.0.4.15
			com_github_llamalad7_mixinextras: MixinExtras 0.2.0-beta.1
		chisel: Chisel Refabricated 1.4.5
			continuity: Continuity 2.0.2+1.19.3-build.2
		cloth-config: Cloth Config v9 9.0.94
			cloth-basic-math: cloth-basic-math 0.6.1
		clumps: Clumps 9.0.0+16
		clutter: Clutter 0.1.5
		collective: Collective 6.52
		colorfulenchantments: Colorful Enchantments Mod 1.1.1+fabric_1.19.3
		comforts: Comforts 6.1.1+1.19.3
		configured: Configured 2.0.2
		contact: Contact 1.1.1-Beta
			midnightlib: MidnightLib 1.1.0
		controlling: Controlling For Fabric 10.0+10
		cookingforblockheads: Cooking for Blockheads 14.0.2
		couplings: Couplings 1.9.2+1.19.3
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
		creativecore: CreativeCore (Fabric) 2.10.4
			net_minecraftforge_eventbus: eventbus 6.0.4
		croptopia: Croptopia 2.2.2
			com_typesafe_config: config 1.4.1
			io_leangen_geantyref_geantyref: geantyref 1.3.11
			org_spongepowered_configurate-core: configurate-core 4.1.2
			org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		customizableelytra: Customizable Elytra 1.6.5-1.19.3
		darkpaintings: DarkPaintings 14.0.2
		darkrtp: DarkRTP - RTP Teleport 1.0.3
		deathbackup: Death Backup 3.0
		dynview: Dynamic View Mod 1.19-3.2
		easyelytratakeoff: Easy Elytra Takeoff 4.0
		ecs: Elytra/Chestplate Swapper 1.3.0-MC1.19
		editsign: EditSign 2.7.0
		elytraslot: Elytra Slot 6.2.0+1.19.3
		enchdesc: EnchantmentDescriptions 14.0.4
		endertanks: EnderTanks 1.0.1
		fab: Foodables 1.0.1a-1.19.3
		fabric-api: Fabric API 0.75.1+1.19.3
			fabric-api-base: Fabric API Base 0.4.21+70be179c85
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.20+49abcf7e85
			fabric-biome-api-v1: Fabric Biome API (v1) 12.1.1+b5d379b085
			fabric-block-api-v1: Fabric Block API (v1) 1.0.5+e022e5d185
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.31+928d44d585
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.12+1134c5b885
			fabric-command-api-v1: Fabric Command API (v1) 1.2.22+f71b366f85
			fabric-command-api-v2: Fabric Command API (v2) 2.2.1+3fc4752e85
			fabric-commands-v0: Fabric Commands (v0) 0.2.39+df3654b385
			fabric-containers-v0: Fabric Containers (v0) 0.1.47+df3654b385
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.5.1+1d37d50285
			fabric-convention-tags-v1: Fabric Convention Tags 1.2.2+8e4e694f85
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.14+aeb40ebe85
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 11.3.1+40dc7a9485
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.42+00a2eb1085
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.7+b83334a085
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.40+3baeb27a85
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.44+df3654b385
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.30+99f9db8085
			fabric-item-api-v1: Fabric Item API (v1) 2.1.10+312c329485
			fabric-item-group-api-v1: Fabric Item Group API (v1) 2.1.12+1134c5b885
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.31+bc01e09785
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.29+df3654b385
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.10+23a79c8a85
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.20+75e9821185
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.24+9e7660c685
			fabric-message-api-v1: Fabric Message API (v1) 5.1.0+c85585f885
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.31+49abcf7e85
			fabric-models-v0: Fabric Models (v0) 0.3.27+11ba9c3b85
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.17+4017a8cb85
			fabric-networking-v0: Fabric Networking (v0) 0.3.34+df3654b385
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 5.4.1+eb2a3ba985
			fabric-particles-v1: Fabric Particles (v1) 1.0.20+f1e4495b85
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.1+5176f73d85
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.1.0+f7b4d36485
			fabric-renderer-api-v1: Fabric Renderer API (v1) 2.2.2+3a95925a85
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.8.0+3a95925a85
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.30+df3654b385
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.25+afca2f3e85
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.18+f1e4495b85
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.33+df3654b385
			fabric-rendering-v1: Fabric Rendering (v1) 1.12.1+eb2a3ba985
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.0+e63306e085
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.10.5+a91e48b785
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.41+f1e4495b85
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.13+99f9db8085
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.8+75e9821185
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.15+ccd377ba85
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 2.3.1+40dc7a9485
		fabric-language-kotlin: Fabric Language Kotlin 1.9.1+kotlin.1.8.10
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.8.10
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.8.10
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.8.10
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.8.10
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.19.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.4.1
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.4.1
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.4.1
		fabricloader: Fabric Loader 0.14.14
		farmingforblockheads: Farming for Blockheads 12.1.0
		floralench: Floral Enchantment Fabric-1.19.3-1.1.2
		forgeconfigapiport: Forge Config API Port 5.0.9
		forgero: Forgero for Fabric 0.10.7-BETA-1.19.3
			advanced_runtime_resource_pack: Runtime Resource Pack 0.6.7
			forgero-common: Forgero common library 0.10.7-BETA-1.19.3
			forgero-compat: Forgero mod compatibility pack 0.10.7-BETA-1.19.3
			forgero-core: Forgero core 0.10.7-BETA-1.19.3
			forgero-extended: Forgero extended pack 0.10.7-BETA-1.19.3
			forgero-fabric-compat: Forgero Fabric compatibility pack 0.10.7-BETA-1.19.3
			forgero-structures: Forgero structure pack 0.10.7-BETA-1.19.3
			forgero-vanilla: Forgero vanilla pack 0.10.7-BETA-1.19.3
		friendsandfoes: Friends&Foes 1.8.0
		fwaystones: Fabric Waystones 3.0.7+mc1.19.3
		geckolib: Geckolib 4.0.4
			com_eliotlash_mclib_mclib: mclib 20
		goblintraders: Goblin Traders 1.8.1
		handcrafted: Handcrafted 2.1.0
		image2map: Image2Map 0.4.0
			map-canvas-api: Map Canvas API 0.2.1+1.19.3
		indium: Indium 1.0.14+mc1.19.3
		inventorysorter: Inventory Sorter 1.8.10-1.19.3
			kyrptconfig: Kyrpt Config 1.5.1-1.19.3
		iris: Iris 1.5.2
			io_github_douira_glsl-transformer: glsl-transformer 2.0.0-pre9
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.11.1
		jamlib: JamLib 0.6.0+1.19.3
		java: OpenJDK 64-Bit Server VM 17
		justmobheads: Just Mob Heads 7.1
		justplayerheads: Just Player Heads 3.1
		lithium: Lithium 0.11.0
		lovely_snails: Lovely Snails 1.1.1+1.19.3
		mcpitanlibarch: MCPitanLib 1.4.9-1.19.3-fabric
		mcwbridges: Macaw's Bridges 2.0.6
		mcwdoors: Macaw's Doors 1.0.8
		mcwfences: Macaw's Fences and Walls 1.0.7
		mcwpaths: Macaw's Paths and Pavings 1.0.2
		mcwroofs: Macaw's Roofs 2.2.3
		mcwtrpdoors: Macaw's Trapdoors 1.0.9
		mcwwindows: Macaw's Windows 2.1.2
		memoryleakfix: memory Leak Fix 1.19.3-0.8.0
		minecraft: Minecraft 1.19.3
		mo_glass: Mo Glass 1.6.1-MC1.19.3
		mob_catcher: Mob Catcher 1.3.7
		modmenu: Mod Menu 5.0.2
		more_slabs_stairs_and_walls: More Slabs, Stairs, and Walls 2.3.0+1.19.3
		morediscs: More Music Discs 1.19.3-27
		moreflowerpots: com.eternalhelldevs.moreflowerpots 3.0-SNAPSHOT
		morefrogs: More Frogs 1.19.3-1.2.5-fabric
		moremobvariants: More Mob Variants 1.0.2
		mousetweaks: Mouse Tweaks 2.24
		myloot: myLoot 3.2-1.19.3
			omega-config: OmegaConfig 1.2.3-1.18.1
		naturescompass: Nature's Compass 1.19.3-2.1.1-fabric
		notenoughanimations: NotEnoughAnimations 1.6.2
		notes: Notes 1.19.3-2.0.0-fabric
		npcvariety: NPC Variety 2.3.0
		numismatic-overhaul: Numismatic Overhaul 0.2.8+1.19.3
			stacc: Stacc 1.3.5
		observable: Observable 4.3.1
		openpartiesandclaims: Open Parties and Claims 0.17.0
		owo: oωo 0.10.2+1.19.3
			blue_endless_jankson: jankson 1.2.1
		patchouli: Patchouli 1.19.3-78-FABRIC
			fiber: fiber 0.23.0-2
		pathunderfencegates: Path Under Fence Gates 1.3.0
		pfm: Paladin's Furniture 1.1.2
		phosphor: Phosphor 0.8.1
		pipeplus: PipePlus 1.19.3-0.5.1
			libblockattributes: LibBlockAttributes 0.12.0
				libblockattributes_core: LibBlockAttributes (Core) 0.12.0
				libblockattributes_fluids: LibBlockAttributes (Fluids) 0.12.0
				libblockattributes_items: LibBlockAttributes (Items) 0.12.0
			libmultipart: LibMultiPart 0.9.0
			libnetworkstack: Lib Network Stack 0.8.0
		polymer-bundled: Polymer (Bundled) 0.3.13+1.19.3
			polymer-blocks: Polymer (Textured Blocks) 0.3.13+1.19.3
			polymer-common: Polymer (Common) 0.3.13+1.19.3
				packet_tweaker: Packet Tweaker 0.3.0+1.18.2
			polymer-core: Polymer (Core) 0.3.13+1.19.3
				polymer-networking: Polymer (Networking) 0.3.13+1.19.3
				polymer-registry-sync-manipulator: Polymer (Registry Sync Manipulator) 0.3.13+1.19.3
			polymer-resource-pack: Polymer (Resource Pack) 0.3.13+1.19.3
		reborncore: Reborn Core 5.5.0
			team_reborn_energy: Energy 2.3.0
		renameit: RenameIt 1.4.0+1.19.x
			fabric-permissions-api-v0: fabric-permissions-api 0.2-SNAPSHOT
			server_translations_api: Server Translations API 1.4.19+1.19.3
		resourcefullib: Resourceful Lib 1.2.7
			com_teamresourceful_yabn: yabn 1.0.2
		rightclickharvest: Right Click Harvest 3.1.0+1.19.3
		roughly_enough_trades: Roughly Enough Trades 1.19.3-1.0
		roughlyenoughitems: Roughly Enough Items 10.0.592
			error_notifier: Error Notifier 1.0.9
		shetiphiancore: ShetiPhianCore 1.3.8
		showmeyourskin: Show Me Your Skin! 1.3.7+1.19.3
			cicada: CICADA 0.1.1
				langfiles-plus: Langfiles Plus 1.0.0
		silencemobs: Silence Mobs 3.0
		silveroak: Silveroak Outpost 1.0.5-Alpha
		simple_backpack: SimpleBackpack 1.4.7
		simple_pipes_dep_container: SimplePipeTest (Dependency Container) 0.9.0
			simple_pipes: SimplePipeTest 0.9.0
		simplehats: SimpleHats 1.19.3-0.1.7
		sodium: Sodium 0.4.9+build.23
		spawnermod: Enhanced Mob Spawners 1.2.3
		spoornpacks: SpoornPacks 4.4-1.19.3
			com_fasterxml_jackson_core_jackson-annotations: jackson-annotations 2.13.1
			com_fasterxml_jackson_core_jackson-core: jackson-core 2.13.1
			com_fasterxml_jackson_core_jackson-databind: jackson-databind 2.13.1
		starterkit: Starter Kit 5.2
		styled-nicknames: Styled Nicknames 1.2.0+1.19.3
		styledchat: Styled Chat 2.1.3+1.19.3
			placeholder-api: Placeholder API 2.0.0-rc.1+1.19.3
			player-data-api: Player Data API 0.2.2+1.19.3
			predicate-api: Predicate API 0.1.0+1.19.3
		styledplayerlist: Styled Player List 2.3.0+1.19.3
		techreborn: Tech Reborn 5.5.0
		terralith: Terralith 2.3
		timeoutout: TimeOut Out 1.0.3+1.19.1
		treeharvester: Tree Harvester 8.0
		trinkets: Trinkets 3.5.1
		universal-graves: Universal Graves 2.2.0+1.19.3
			common-protection-api: Common Protection API 1.0.0
			hologram-api: Hologram API 0.2.3+1.19.3
			sgui: sgui 1.2.0+1.19.3
		variant_bookshelves: Variant Bookshelves 1.4
		variantcraftingtables: Variant Crafting Tables 4.1
		veinmining: Vein Mining 1.1.1+1.19.3
			spectrelib: SpectreLib 0.12.3+1.19.2
		villagernames: Villager Names 4.5.1
		villagersplus: Villagers Plus 1.8
		vmp: Very Many Players 0.2.0+beta.7.45
		wi_zoom: WI Zoom 1.4-MC1.19.3
		worldedit: WorldEdit 7.2.13+46576cc
		wthit: wthit 6.2.0
		xaerominimap: Xaero's Minimap 23.2.0
		xaeroworldmap: Xaero's World Map 1.29.2
		xercamusic: Music Maker Mod fabric-1.19.3-1.0.0
		xercapaint: Joy of Painting fabric-1.19.3-1.0.0
	Loaded Shaderpack: (off) (fallback)
	NEC status: No NEC detected
	Launched Version: fabric-loader-0.14.14-1.19.3
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 1070 Ti/PCIe/SSE2 GL version 3.2.0 NVIDIA 526.86, NVIDIA Corporation
	Window size: 1920x1080
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: fabric
	Current Language: English (US)
	CPU: 12x AMD Ryzen 5 3600 6-Core Processor
```

### Additional context

I'm not very good with stuff like this so I may need it to be closer to an ELI5 (explain like im five). Thanks guys!

**Comment by douira** [comment]
Does this still happen? If it does, please try to narrow it down to the smallest possible set of mods the can reproduce the crash. If it doesn't, this issue can be closed.
